src/app/item-popup/DesktopItemActions.tsx

```diff
@@ -98,10 +71,7 @@ export default function DesktopItemActions({ item }: { item: DimItem }) {
   useLayoutEffect(() => {
     const reposition = () => {
       if (containerRef.current) {
-        let parent = containerRef.current.parentElement;
-        while (parent && !parent.classList.contains('item-popup')) {
-          parent = parent.parentElement;
-        }
+        const parent = containerRef.current.closest('.item-popup');
```

Comment:
I didn't mean to sneak this change in here, but it does work as expected

Reply:
Whoa, I didn't know about `closest`!

Reply:
support seems pretty decent, no IE at all though... I'm not even sure what DIM technically can go back to..